### DISCUSSION TITLE
What is this iconBlank all about?

### DISCUSSION THREAD

**Discussion by loxy** [taiga-family/taiga-ui#2768] [Q&A] [answered]
Hi!

What is this `iconBlank` all about?

I'm asking because we have annoying console warnings on many tests:

"WARNING: sanitizing HTML stripped some content, see https://g.co/ng/security#xss"

And they come all from this special icon: 

```
<svg xmlns="http://www.w3.org/2000/svg"><g id="iconBlank" xmlns="http://www.w3.org/2000/svg"><svg></svg></g></svg>
```

This is the output of the `unsafeHtml` in `_sanitizeHtml` in `core.mjs` (Angular)

Do you know how to get rid of this?

Best regards,
Kersten

**Comment by waterplea** [comment, marked as answer]
It's a pretty hacky approach we used to reserve space for icon without actually showing one. You can file an issue so we come up with something else, it is only used in one place and I'm sure there's a better way to solve this 😁

**Reply by loxy** [reply]
Thank you. Here the feature request: https://github.com/Tinkoff/taiga-ui/issues/2779
